packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -134,7 +133,7 @@ export const useNotifiSubscribe: ({
     // If there is a target, set the discord target data to it, otherwise create a new target.
     setDiscordTargetData(
```

Comment:
I think i mentioned this previously, but I think it's probably better if we make this a `useState<DiscordTarget | undefined>` and use `undefined` value to have the semantic meaning of "no target available" rather than having a placeholder with fake data.

Reply:
ooo i have to address that change. let me update the logic